frontend/packages/dev-console/src/components/pipelines/pipeline-topology/BuilderNode.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import { Tooltip } from '@patternfly/react-core';
+import { ExclamationIcon, PlusIcon } from '@patternfly/react-icons';
+import { Node, observer } from '@console/topology';
+import { AddNodeDirection, BUILDER_NODE_ADD_RADIUS, BUILDER_NODE_ERROR_RADIUS } from './const';
+import TaskNode from './TaskNode';
+import { BuilderNodeModelData } from './types';
+
+import './BuilderNode.scss';
+
+const drawAdd = (x, y, onClick) => {
+  return (
+    <g className="odc-builder-node__add-icon" onClick={onClick} transform={`translate(${x}, ${y})`}>
+      <circle cx={0} cy={0} r={BUILDER_NODE_ADD_RADIUS} fill="blue" />
```

Comment:
Should use PF colors for the blue.

Reply:
Good catch, was cleaning the other colours up and missed this one.